.travis.yml

```diff
@@ -27,6 +29,8 @@ matrix:
       env: WP_VERSION=latest
     - php: 5.6
       env: WP_VERSION=trunk
+    - php: 5.4
```

Comment:
Travis complains with 
                                            
  [Composer\Downloader\TransportException]  
  Peer fingerprint did not match

only for the PHP 5.4 test version

Looks like a Composer setup issue?